media/js/app/assetmgr/asset.js

```diff
@@ -733,6 +749,11 @@
         };
 
         this.showAnnotation = function(event, ui) {
+            if (document.forms['annotation-list-filter']) {
+                document.forms['annotation-list-filter']
+                    .elements.showall.checked = false;
```

Comment:
I'm not sure about setting the `showall` to false here. Currently, the image viewer continues to show all selections, and simply focuses on the selected one. (Image also has different colored borders, but we can leave that for another time.)

Reply:
Alright, the showall behavior should be closer to how it works on the Image type, now.